PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+(18:11) the final say over consensus but relies on miners to order transactions and developers to create new updates that both the miners and the node network accept many other blockchains make multiple trade-offs including making the nodes require immense processing power bandwidth and storage so that only large entities can run them which centralizes the network into a handful of major providers the point i want to make here is that this is also slippery slope right of course like increasing the requirements by 10 maybe ethereum increases by 10 a
+
+(18:44) year can have an outsized effect where suddenly 90 of the people who would have done this can no longer do it right so it's not linear yeah and once you start going down that path right the reasonable central slope is you're like well you know it's already hard so it's not a big deal we make it a little bit harder and this is why the thought of like increasing the bitcoin block size is so risky so increasing the complexity on the base layer of a blockchain makes it harder for an individual to run a full node
+
+(19:11) and without full nodes the blockchain relies more on trust and much less on decentralization bitcoin's proof of work and small block design keep a lot of power with the individual users the block size war in 2017 was a real world test of bitcoin's level of decentralization ever since the network's early history there was a growing divide between people who wanted to increase the block size and people who wanted to keep it small increasing the block size allows the network to process more transactions however
+
+(19:43) increasing the block size also increases the bandwidth and data storage required to run a full node and in return it puts it out of reach for the everyday user at one point in 2017 over 80 percent of minor processing power the biggest maker of bitcoin mining equipment and a large number of major custodians and exchanges including coinbase and grayscale were in favor of increasing the block size with an upgrade called segwit2x and yet they failed why did it fail because the plebes were against the idea that's
+
+(20:20) only possible in a network where you have so many people running nodes at home uh and threatening to do a uasf or something if if the change proceeds and that i think realistically that would be a very um difficult thing for ethereum to achieve right now because not many people run the full nodes if bitcoin were built on a proof of stake model where the more coins you have the more votes you have on how the network functions the large exchanges and custodians could have used the millions of coins they held on clients
```

Comment:
Distinguere le frasi non del narratore `why did it fail because the plebes were against the idea that's [...] difficult thing for ethereum to achieve right now because not many people run the full nodes`